README.md

```diff
@@ -1,7 +1,6 @@
-# Shaarli app for Yunohost
-=======
-shaarli_ynh - Shaarli app for Yunohost
-===========
+# Shaarli app for Yunohost (shaarli_ynh)
+---
+
```

Comment:
Shaarli app for Yunohost ~~(shaarli_ynh)~~
Its better to have short heading. Shaarli_ynh has no information for the end user.